plugins/plugin-chart-echarts/src/Pie/controlPanel.tsx

```diff
@@ -147,10 +154,23 @@ const config: ControlPanelConfig = {
               min: 10,
               max: 100,
               step: 1,
-              default: 50,
+              default: outerRadius,
               description: t('Outer edge of Pie chart'),
             },
           },
+          noopControl,
+        ],
+        [
+          {
+            name: 'donut',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Donut'),
+              default: donut,
+              renderTrigger: true,
+              description: t('Do you want a donut or a pie?'),
```

Comment:
Jeez... asking the hard questions here, man!

Reply:
At some point, it might be good to make a custom "switch" component for when checkboxes just won't do:

![donutpie](https://user-images.githubusercontent.com/812905/101439724-faaa9f80-38c9-11eb-8ee2-fe99ebcdd778.gif)